lib/index.js

```diff
@@ -0,0 +1,91 @@
+const fs = require('fs');
+const inquirer = require('inquirer');
+const config = require('./config');
+
+const getStoryPrompt = () => {
+  return [
+    {
+      choices: config.stories.map(story => {
+        return {
+          name: story.title,
+          value: story.id
+        };
+      }),
+      message: config.welcome,
+      name: 'story',
+      type: 'list'
+    }
+  ];
+};
+
+const getWordsPrompt = story => {
+  const template = getStoryTemplate(story.template);
+  const words = getWordsFromStoryTemplate(template);
+
+  const prompt = words.map(word => {
+    const conf = config.words.find(configWord => configWord.id === word);
+    const message = conf ? conf.question : `Enter a ${word}:`;
+
+    return {
+      message,
+      name: word,
+      type: 'input'
+    };
+  });
+
+  return prompt || [];
+};
+
+const getStoryTemplate = template => {
+  return fs.readFileSync(`${__dirname}/stories/${template}`).toString();
+};
+
+const getWordsFromStoryTemplate = story => {
+  /**
+   * UNIT TEST NOTES: if `getWordsFromStory` was simply...
+   *
+   * return story.match(/{{[{]?(.*?)[}]?}}/g).map(word => word.replace(/({{)|(}})/g, ''));
+   *
+   * ... it would throw a `TypeError` if the story did not contain any variables. maybe
+   * we use the line of code above to start with, and then our unit tests will help us
+   * catch the bug and we refactor to the version below.
+   *
+   */
+
+  const variables = story.match(/{{[{]?(.*?)[}]?}}/g);
```

Comment:
we should assume they won't know the regex here so a comment about what it will accept would be good.